### ISSUE TITLE
libad9361-iio-0.1: fatal: Not a git repository

### ISSUE THREAD

**Issue by blshkv** [analogdevicesinc/libad9361-iio#23]
I'm trying to compile a stable package released recently. It fails with the following error:
```
 * Package:    net-libs/libad9361-iio-0.1
 * Repository: pentoo
 * USE:        abi_x86_64 amd64 elibc_glibc kernel_linux userland_GNU
 * FEATURES:   preserve-libs sandbox userpriv usersandbox
>>> Unpacking source...
>>> Unpacking libad9361-iio-0.1.tar.gz to /var/tmp/portage/net-libs/libad9361-iio-0.1/work
>>> Source unpacked in /var/tmp/portage/net-libs/libad9361-iio-0.1/work
>>> Preparing source in /var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1 ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1 ...
>>> Working in BUILD_DIR: "/var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1_build"
cmake -C /var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1_build/gentoo_common_config.cmake -G Unix Makefiles -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Gentoo -DCMAKE_USER_MAKE_RULES_OVERRIDE=/var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1_build/gentoo_rules.cmake -DCMAKE_TOOLCHAIN_FILE=/var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1_build/gentoo_toolchain.cmake  /var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1
loading initial cache file /var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1_build/gentoo_common_config.cmake
-- The C compiler identification is GNU 7.3.0
-- Check for working C compiler: /usr/bin/x86_64-pc-linux-gnu-gcc
-- Check for working C compiler: /usr/bin/x86_64-pc-linux-gnu-gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Found Git: /usr/bin/git (found version "2.16.4") 
fatal: Not a git repository (or any parent up to mount point /var/tmp)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
CMake Error at CMakeLists.txt:35 (if):
  if given arguments:

    "STREQUAL" "/var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1"

  Unknown arguments specified


-- Configuring incomplete, errors occurred!
See also "/var/tmp/portage/net-libs/libad9361-iio-0.1/work/libad9361-iio-0.1_build/CMakeFiles/CMakeOutput.log".
```

**Comment by lclausen-adi** [comment]
Hi,

Thanks for reporting this. It should be fixed with 7206bb2a9b655be3bdb66c6cf03aa504817ed240.

**Comment by blshkv** [comment]
it is fixed indeed, thanks!
